test/unit/model/include.test.js

```diff
@@ -405,6 +405,26 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         expect(options.include[0].subQueryFilter).to.equal(false);
       });
 
+      it('should not tag a separate hasMany association with subQuery true', function() {
```

Comment:
In addition to this test, please add an integration test [here](https://github.com/sequelize/sequelize/blob/master/test/integration/include/separate.test.js)

Reply:
Added integration test